### PR TITLE
s4583222 - Oasis Brain Dataset (Stable Diffusion)

### PR DESCRIPTION
**Pull request by s4583222**
Name: Thomas Jellett
Student Number: 45832220
(Written in PyTorch)

Hi everyone,
the folder recognition/s4583222_Solution includes:

README.md: Report on the problem and the solution
train.py: Contains the code for training, saving the model, and plotting the loss
dataset.py: Contains functions to generate the dataloader for the oasis brain dataset
module.py: Contains the modified UNET/Stable Diffusion model
predict.py: Contains the code to load the model and generate images with it

As seen from my README.md, my implementation of stable diffusion produces ("reasonably clear") results that are similar to the Oasis Brain dataset. The README.md gives instructions on how to reproduce the results.
Thanks,
Tom

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
**This is an initial inspection, no action is required at this point**

- Readme: OK, no need to do a full dependency dump, list only the key ones in the future.
- Results: Acceptable. Can see the model is doing something, loss is decreasing also.
- Code: i remeber the loss being MSE instead of the MAE used here, though i doubt it would make a massive difference.
- Commit messages: OK but some should be more specific

IMPORTANT: Commits are not in the right place. The source branch should not be master.
 You should fork Shakes repo in whole (there is a checkmark when you fork) and commit to his topic-recognition branch. You should be able to see work from the previous years in "recognition folder"

**Comment by s4583222** [comment]
Hello,
I accidentally deleted my PR trying to fix this.
Help!
________________________________
